app/models/sciencewire_source_record.rb

```diff
@@ -93,11 +93,13 @@ def self.get_sciencewire_source_record_for_pmid(pmid)
   end
 
   def self.get_sciencewire_source_record_from_sciencewire(pmid)
+    return nil unless Settings.SCIENCEWIRE.enabled
```

Comment:
These method should be **removed** as in #850.  We're never enabling Sciencewire again, so gating on a setting suggests functionality that doesn't exist.